ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -1504,13 +1507,15 @@ internal sealed class ArchiWebHandler : IDisposable {
 
 				uint realAppID = appID;
 				Steam.Asset.EType type = Steam.Asset.EType.Unknown;
+                bool Marketable = false;
```

Comment:
I think logic for "Marketable" should always be treated as True unless it's been confirmed otherwise. In the case of some awful bug, you do not want it to treat some actual marketable card as cross-set.

Reply:
Makes sense. This way it will be safer.